scripts/supervisor.sh

```diff
@@ -88,6 +88,21 @@ function init_udev() {
     udevadm trigger && udevadm settle
 }
 
+function init_os-agent() {
+    if pgrep os-agent; then
+        echo "os-agent is running"
+        return 0
+    fi
+
+    if [ ! -f /usr/sbin/os-agent ]; then
+        curl -Lo /usr/sbin/os-agent https://github.com/home-assistant/os-agent/releases/download/1.1.0/os-agent-debian-amd64.bin
```

Comment:
```suggestion
        curl -Lo /usr/sbin/os-agent https://github.com/home-assistant/os-agent/releases/latest/download/os-agent-debian-amd64.bin
```
Let's always use the latest tag so we don't have to update this file on each release